hack/tools/vault_utils.sh

```diff
@@ -50,6 +49,7 @@ enable_k8s_auth() {
         kubectl get secret $2 -n $3 -o jsonpath="{.data['ca\.crt']}" | base64 --decode > tmp/ca.crt
 
         # Configure the k8s sa auth
+        # TODO: Add issuer for kubernetes versions greater than 1.20
```

Comment:
Is that still a TODO? I thought that's what post-pod-creation-cm.yaml does

Reply:
Yes it is still a TODO. we currently have two kinds of values.yaml for vault deployments:
1) vault-single-cluster-values.yaml - single cluster setup. used in quick start. it includes vault configuration of auth methods, policies and etc
2) plugin-secrets-kubernetes-reader-values.yaml - deployment of vault without vault configuration of auth methods, policies etc (only loading the plugin).  it is now used for testing including for multi-cluster setup. vault_utils.sh  script is used to configure vault after its deployment and thus the change in vault-single-cluster-values.yaml which is in this pr is not applied to this case.